cmd/plugin/main.go

```diff
@@ -51,6 +51,8 @@ var (
 	enableCache = flag.Bool("enable-daemon-image-credential-cache", true,
 		"Whether to save contents of imagepullsecrets of the daemon ServiceAccount in memory. "+
 			"If set to false, secrets will be fetched from the API server on every image pull.")
+	asyncImagePulls = flag.Bool("async-image-pulls", false,
```

Comment:
are you planning to support other parameters of async mode later ?

Reply:
That's the plan. Yes.